Nos/Service/Analytics.swift

```diff
@@ -30,7 +30,7 @@ class Analytics {
         track("Published Note", properties: ["length": note.content.count])
     }
 
-    func published(reply: JSONEvent) {
+    func published(reply: JSONEvent) { // TODO: delete? it's unused
```

Comment:
@mplorentz I'd love your thoughts on these two as well. Is there any reason to keep them since we probably tracked these events in analytics at some point?

Reply:
oh 🤔 we should be firing this. I'm just going to fix it in another PR.